### PR TITLE
Take action page bare bones

### LINKED ISSUE #191: Bare Bones Take Action Page
## Description

The Bare Bones take action page is simple: It will have one section (no in-page nav bar): An Open Letter. The letter will be stored in contentful, and will be in the richtext format

This is the MVP of the Take Action Page.

Image

Image

Image

## Todo

- [x] Create Take Action page
- [x] Create contentful model for Open Letter & add the letter to contentful @kennykos

### PR DESCRIPTION
**Pull request by lizageorge**
## Summary

Closes #191 

## Changes

- Created take-action page
- Added navbars
- Added sections with headers
- Created Contentful model for open letter, imported to OpenLetter section

## Screenshots
image
image

### CODE REVIEW

File: src/components/navigation/PrimaryNavBar.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ const PrimaryNavBar: FC = () => {
         <Navbar.Toggle />
         <Navbar.Collapse>
           <NavButton title="Home" link="/home" />
+          <NavButton title="Take Action" link="/open-letter" />
```
Comment (kennykos) [inline src/components/navigation/PrimaryNavBar.tsx:51]: When I click on the nav-bar I get directed to `http://localhost:3000/open-letter` (which is good) but the page gives a `404` error.

image

We should be routing to `take-action`

**Review by kennykos — APPROVED** [review]
Looks good to me! Great Job :smile:

File: src/sections/take-action/letter.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div className="flex flex-col items-center">
+      <div>
+        <HighlightedTitle
+          title="An Open Letter"
+          size="large"
+          color="clementine"
+        />
+      </div>
+      <div className="space-y-4 px-60">
```
Comment (kennykos) [inline src/sections/take-action/letter.tsx:22]: I know this wasn't in the Figma and Issue, but looking at the page, the text seems to be too wide.
image

We should be aiming for text of more paper width, like so 
image
https://fossilfreeresearch.org/letter/

**Review by kennykos — CHANGES_REQUESTED** [review]
One small change, but otherwise this looks very good to me!

File: src/sections/take-action/letter.tsx (hunk 2 above)
Comment (kennykos) [inline src/sections/take-action/letter.tsx:22]:
```suggestion
      <div className="space-y-6 w-2/4">
```

This spacing should do the trick, after that it should be good to go!

**Review by kennykos — CHANGES_REQUESTED** [review]
Ok, with this spacing changes this commit should be good to go and ready to merge.

File: src/sections/take-action/letter.tsx (hunk 2 above)
Comment (lizageorge) [reply src/sections/take-action/letter.tsx]: ok, updated this! but it's kind of a temporary fix, we should 100% create a more mobile-friendly option before quad day.

**Review by kennykos — APPROVED** [review]
Looks good to me!

**Pull request merged**